### DISCUSSION TITLE
Existing project does not show up at "Reporting > Project overview"

### DISCUSSION THREAD

**Discussion by BabyFnord** [kimai/kimai#5064] [Questions & Answers] [answered]
### Describe the issue

Running a self-hosted Kimai I experience this glitch:
A project listed on the `Administration > Projects` page does not show up at "Reporting > Project overview". Comparing the project settings, I see no real differences between them. Maybe it's just me, maybe something went wrong the last time I upgraded Kimai?

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.21.0

### How do you run Kimai?

Shared-Hosting

### Which PHP version are you using?

Other (please mention below)

### Logfile

`PHP 8.3.8`

### Screenshots

![Bildschirmfoto 2024-09-13 um 12 31 36](https://github.com/user-attachments/assets/e987a36c-cf9b-4855-a2a4-56d64b4a08cd)
![Bildschirmfoto 2024-09-13 um 12 33 26](https://github.com/user-attachments/assets/a7c308e7-4811-4f02-806e-6a5860840f8a)

**Comment by kevinpapst** [comment, marked as answer]
You mean the one with the yellow background? Projects that are over or hidden will be excluded from the reporting.

**Reply by BabyFnord** [reply]
Doh 😃 … thanks @kevinpapst!
